### ISSUE TITLE
CRC 1.27 fails to start VM - Unable to determine VM's IP address

### ISSUE THREAD

**Issue by jmajulian21** [code-ready/crc#2397]
### General information

  * OS: Linux
  * Hypervisor: KVM
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on:  VM

## CRC version
```bash
# Put `crc version` output here
CodeReady Containers version: 1.27.0+3d6bc39d
OpenShift version: 4.7.11 (embedded in executable)

## CRC config
# Put `crc config view` output here
- consent-telemetry                     : yes


## Host Operating System
NAME="Red Hat Enterprise Linux"
VERSION="8.4 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.4"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.4 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8.4:GA"
HOME_URL="https://www.redhat.com/"
DOCUMENTATION_URL="https://access.redhat.com/documentation/red_hat_enterprise_linux/8/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.4
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.4"

### Steps to reproduce

  1. crc setup
  2. crc start

### Expected
State running without errors

### Actual
State running with errors

### Logs
INFO Starting CodeReady Containers VM for OpenShift 4.7.11...
DEBU Updating CRC VM configuration
DEBU (crc) Calling .GetConfigRaw
DEBU (crc) Calling .Start
DEBU (crc) DBG | time="2021-06-02T15:43:55-03:00" level=debug msg="Starting VM crc"
DEBU (crc) DBG | time="2021-06-02T15:43:55-03:00" level=debug msg="Validating network"
DEBU (crc) DBG | time="2021-06-02T15:43:55-03:00" level=debug msg="Validating storage pool"
DEBU (crc) DBG | time="2021-06-02T15:44:00-03:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-06-02T15:44:00-03:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-06-02T15:44:00-03:00" level=debug msg="Waiting for machine to come up 0/60"
DEBU (crc) DBG | time="2021-06-02T15:44:03-03:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-06-02T15:44:03-03:00" level=debug msg="Getting current state..."
DEBU (crc) DBG | time="2021-06-02T15:44:03-03:00" level=debug msg="Waiting for machine to come up 1/60"
DEBU (crc) DBG | time="2021-06-02T15:44:06-03:00" level=debug msg="GetIP called for crc"
DEBU (crc) DBG | time="2021-06-02T15:44:06-03:00" level=debug msg="Getting current state..."
.
.
DEBU (crc) DBG | time="2021-06-02T15:46:57-03:00" level=debug msg="Waiting for machine to come up 59/60"
DEBU (crc) DBG | time="2021-06-02T15:47:00-03:00" level=warning msg="Unable to determine VM's IP address, did it fail to boot?"
DEBU Making call to close driver server
DEBU (crc) Calling .Close
DEBU Successfully made call to close driver server
DEBU Making call to close connection to plugin binary
DEBU (crc) DBG | time="2021-06-02T15:47:00-03:00" level=debug msg="Closing plugin on server side"
Error starting machine: Error in driver during machine start: Unable to determine VM's IP address, did it fail to boot?

Before gather the logs try following if that fix your issue
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
Can you try running with the user network mode?
https://github.com/code-ready/crc/wiki/VPN-support--with-an--userland-network-stack

**Comment by praveenkumar** [comment]
Check https://github.com/code-ready/crc/issues/2214 if you have recently updated or have docker clients.

**Comment by jmajulian21** [comment]
> Can you try running with the user network mode? Yes, I performed  the steps for Linux and it did not work. (log attached)
> https://github.com/code-ready/crc/wiki/VPN-support--with-an--userland-network-stack

[crc-start.log](https://github.com/code-ready/crc/files/6600635/crc-start.log)
[crc-deamon.log](https://github.com/code-ready/crc/files/6600636/crc-deamon.log)

**Comment by praveenkumar** [comment]
@jmajulian21 This looks like vm is  failed to start, not sure why. Can you check `virsh -c qemu:///system list` after crc start failed? And also look in `journalctl -u libvirtd`, and in `/var/lib/libvirt/qemu/crc.log`

**Comment by jmajulian21** [comment]
> @jmajulian21 This looks like vm is failed to start, not sure why. Can you check `virsh -c qemu:///system list` after crc start failed? And also look in `journalctl -u libvirtd`, and in `/var/lib/libvirt/qemu/crc.log`

virsh -c qemu:///system list
![image](https://user-images.githubusercontent.com/14821311/121086141-2c3cee80-c7b9-11eb-84df-6a6b05d18ee1.png)

journalctl -u libvirtd
![image](https://user-images.githubusercontent.com/14821311/121086324-660df500-c7b9-11eb-9af6-141a8345f7fd.png)

"/var/lib/libvirt/qemu/crc.log": The file does not exists

**Comment by Xplouder** [comment]
Same problem here, cant find a way to workaround this.

**Comment by jmajulian21** [comment]
Do you know another way to make a locally raised openshift? with CetOs or with windows? I need to know urgently :'(

**Comment by Xplouder** [comment]
@jmajulian21 try with a centos vagrant box. I managed to execute the setup and start commands successfully but did didn't work as expected. Next week i might keep debugging this.

**Comment by praveenkumar** [comment]
@jmajulian21 I can see you are running it on a VM can you tell us what is the base OS and hypervisor? In the past we did see some issue with CRC when users tried to run it in nested  https://github.com/code-ready/crc/issues/2082

**Comment by Xplouder** [comment]
@praveenkumar i guess so. Tryed using arch but `libvirt` is not recognized by the binary even when  im sure that it is installed. Then tryed use a centos vagrant box since its a supported distro but faced the problem described in this issue.

**Comment by praveenkumar** [comment]
> @praveenkumar i guess so. Tryed using arch but `libvirt` is not recognized by the binary even when im sure that it is installed. Then tryed use a centos vagrant box since its a supported distro but faced the problem described in this issue.

@Xplouder centos vagrant box directly on the host which have arch?

**Comment by Xplouder** [comment]
@praveenkumar yes, Manjaro to be more precise.

**Comment by johnwmail** [comment]
Hi, any workaround? Thanks.

**Comment by praveenkumar** [comment]
@johnwmail This is a closed issue, can you create a new issue using template and provide us more info around it?
